ci/tests/general/general_system.py

```diff
@@ -35,9 +35,9 @@ class GeneralSystem(object):
     @staticmethod
     def list_ovs_services(host=IP):
         if GeneralSystem.INIT_SYSTEM == 'init':
-            return General.execute_command_on_node(host, ['initctl', 'list', '|grep', 'ovs-*']).splitlines()
+            return General.execute_command_on_node(host, ['initctl list | grep ovs-*'], allow_insecure=True).splitlines()
```

Comment:
allow_insecure flag makes it expect a string, not a list.
Line should be :
General.execute_command_on_node(host, 'initctl list | grep ovs-*', allow_insecure=True).splitlines()